test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -93,11 +91,11 @@ public void addPerson_emptyAddressBook() throws Exception {
 
     @Test
     public void addPerson_someTagsNotInTagList() throws Exception {
-        assertFalse(defaultAddressBook.containsTag(tagEconomist));
-        assertFalse(defaultAddressBook.containsTag(tagPrizeWinner));
+        assertFalse(defaultAddressBook.getAllTags().contains(tagEconomist));
+        assertFalse(defaultAddressBook.getAllTags().contains(tagPrizeWinner));
```

Comment:
@PierceAndy I assume that you are talking about this one in the OP of the PR. 

Since `containsTag()` is only used in testing, and we are removing it, this alternative looks acceptable. However, it seems that we also have a method `AddressBookTest#isTagObjectInAddressBookList()` that also allow us to check the tags in the AddressBook (like how it used in line 100). Maybe consider using that instead?